active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,622 @@
+    <controls:InfoBar.IconSource>
+        <controls:SymbolIconSource Symbol="Phone" />
+    </controls:InfoBar.IconSource>
+</controls:InfoBar>
+```
+
+![A mockup of an InfoBar in the default state with a custom background color and icon](images/Custom_IconColor.png)
+
+## Add an action button
+
+An additional action button can be added by defining your own button that inherits 
+[ButtonBase](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.Primitives.ButtonBase) 
+and setting it in the ActionButton property. Custom styling will be applied to action buttons of type 
+[Button](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Button) 
+and [HyperlinkButton](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.HyperlinkButton) 
+for consistency and accessibility. Outside of the ActionButton property, 
+additional action buttons can be added via custom content and will appear below the message.
+
+
+XAML
+```xml
+<controls:InfoBar x:Name="NoInternetNotification"
+    Severity="Error"
+    Title="No Internet"
+    Message="Reconnect to continue working.">
+    <controls:InfoBar.ActionButton>
+        <Button Content="Network Settings"/>
+    </controls:InfoBar.ActionButton>
+</controls:InfoBar>
+```
+
+![A mockup of an InfoBar in the Error state with a single line message and an action button](images/Error_ActionButton.png)
+
+XAML
+```xml
+<controls:InfoBar x:Name="TermsAndConditionsUpdatedNotification"
+    Title="Terms and Conditions Updated"
+    Message="Dismissal of this message implies agreement to the updated Terms and Conditions. Please view the changes on our website.">
+    <controls:InfoBar.ActionButton>
+        <HyperlinkButton
+            Content="Terms and Conditions Sep 2020" 
+            NavigateUri="http://www.microsoft.com"/>
+    <controls:InfoBar.ActionButton/>
+</controls:InfoBar>
+```
+![A mockup of an InfoBar with a message expanding multiple lines and a hyperlink](images/Default_Hyperlink.png)
+
+## Content wrapping
+The text set in the Message property will automatically wrap vertically in the control 
+underneath the other visual components if the control is no longer able to expand horizontally. 
+The action button will also wrap in its own line.
+
+
+XAML
+```xml
+<controls:InfoBar x:Name="BackupCompleteNotification"
+    Severity="Success"
+    Title="Backup complete: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."  
+    Message="All documents were uploaded successfully. Ultrices sagittis orci a scelerisque. Aliquet risus feugiat in ante metus dictum at tempor commodo. Auctor augue mauris augue neque gravida." />
+```
+
+![A mockup of an InfoBar in the Success state multiline title and message InfoBar](images/Success_ContentWrapping.png)
+
+
+## Custom content
+XAML content can be added to an InfoBar using the Content property. 
+It will appear in its own line below the rest of the control content. 
```

Comment:
Below the `ActionButton` as well?

Reply:
Yep, see below:
![image](https://user-images.githubusercontent.com/20007011/93113016-81563480-f66d-11ea-937b-ec012d1077c6.png)